src/VisualStudio/Core/Def/ErrorReporting/VisualStudioErrorReportingService.cs

```diff
@@ -19,6 +20,7 @@ namespace Microsoft.CodeAnalysis.ErrorReporting
     internal partial class VisualStudioErrorReportingService : IErrorReportingService
     {
         private readonly IThreadingContext _threadingContext;
+        private readonly IVsService<IVsActivityLog> _activityLog;
         private readonly IAsynchronousOperationListener _listener;
         private readonly VisualStudioInfoBar _infoBar;
         private readonly SVsServiceProvider _serviceProvider;
```

Comment:
This appears to be unused now, but could be wrong if GitHub is making it difficult for me.

Reply:
➡️ Amended the change to remove this unused field.